Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM registry.fedoraproject.org/fedora:25
+FROM registry.fedoraproject.org/fedora:27
 MAINTAINER Jonathan Lebon <jlebon@redhat.com>
 
 # NB: we install libyaml-devel so that we can use
 # CSafeLoader in PyYAML (see related comment in the parser)
 
-RUN dnf install -y \
+RUN dnf install --setopt=tsflags=nodocs -y \
```

Comment:
I don't think this is necessary because those options are enabled by default in the `fedora:27` image:

```
$ sudo docker run -it registry.fedoraproject.org/fedora:27 cat /etc/dnf/dnf.conf
[main]
gpgcheck=1
installonly_limit=3
clean_requirements_on_remove=True
tsflags=nodocs
```